woocommerce/woocommerce-android
# [Mobile Payments] Handle enabled status the same way as completed

Closes: #9636


### Description

There is a new status (`enabled`) returned from the API that we do not handle. This PR adds handling of it in the same as `completed`. More context - p1692780778792139-slack-C025A8VV728

Fluxc changes - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2819

### Testing instructions

* As I don't know how to reproduce the status, use proxyman or Charles
* Remap `payments/Faccounts` API call to the response (example):
```
{
  "data": {
    "is_live": false,
    "has_pending_requirements": false,
    "has_overdue_requirements": false,
    "current_deadline": null,
    "status": "enabled",
    "statement_descriptor": "NEW STATEMENT ",
    "store_currencies": {
      "default": "usd",
      "supported": [
        "usd"
      ]
    },
    "country": "US",
    "test_mode": true
  }
}
```
* Go to payments hub
* Notice that error onboarding is in completed state - error is not shown anymore
